website/static/js/logTextParser.js

```diff
@@ -223,7 +224,7 @@ var LogPieces = {
         view: function (ctrl, logObject) {
             var destination = logObject.attributes.params.destination;
             if(paramIsReturned(destination, logObject)){
-                return m('a', {href: destination.url}, destination.materialized);
+                return m('span', [m('a', {href: destination.url}), destination.materialized, ' in ', source.addon]);
```

Comment:
I'm getting a console error pointing to this line -- `Uncaught ReferenceError: source is not defined`